junit-platform-launcher/src/main/java/org/junit/platform/launcher/TagFilter.java

```diff
@@ -86,7 +85,18 @@ public static PostDiscoveryFilter includeTags(String... tagExpressions) throws P
 	 * @see TestTag#isValid(String)
 	 */
 	public static PostDiscoveryFilter includeTags(List<String> tagExpressions) throws PreconditionViolationException {
-		return includeMatching(tagExpressions, Stream::anyMatch);
+
+		return includeMatching(tagExpressions);
+	}
```

Comment:
Ok now I see what you meant in https://github.com/junit-team/junit5/pull/2043#discussion_r357924111... and I think I agree 😅
My bad, I guess I didn't look at it properly...
I mean I still don't really like the way it looked before, but you were right this is no better. So I don't know, if you want to put it back the way it was or if you have another idea, go on 😉 (sorry again)

Reply:
I believe the current code is more readable than the previous one, let's go with this. :P